src/kakarot/model.cairo

```diff
@@ -7,18 +7,36 @@ from starkware.cairo.common.uint256 import Uint256
 from starkware.cairo.common.dict import DictAccess
 
 namespace model {
+
+    // @dev Stack with a 1024 items size. Each item is a 256 bits word. The stack is used by most 
+    // @dev opcodes to consume their parameters from.
+    // @dev Each element is represented by two 128bits (16bytes) chunks.
+    // @param word_dict_start - pointer to a DictAccess array used to simulate a read/write memory
```

Comment:
"memory" is confusing here as there's also the Memory down below.

You can say "DictAccess is used to store the stack's value at a given index" or something like that